### PR TITLE
Fix fileExtensionShown toggle for ResourceTiles

### PR DESCRIPTION
**Pull request by pascalwengerter**
## Description
Random find, worth the fix I suppose

### CODE REVIEW

File: packages/web-app-files/src/components/FilesList/ResourceTiles.vue [hunk 1]
```diff
@@ -287,6 +289,9 @@ export default defineComponent({
     return {
       ImageDimension
     }
+  },
+  computed: {
+    ...mapState('Files', ['areFileExtensionsShown'])
   }
```
Comment (JammingBen) [inline packages/web-app-files/src/components/FilesList/ResourceTiles.vue:295]: You can do that via Composition API as well: `store.getters['Files/areFileExtensionsShown']` 🙂

**Review by JammingBen — CHANGES_REQUESTED** [review]

File: packages/web-app-files/src/components/FilesList/ResourceTiles.vue (hunk 1 above)
Comment (pascalwengerter) [reply packages/web-app-files/src/components/FilesList/ResourceTiles.vue]: Applied, good point

Comment (JammingBen) [reply packages/web-app-files/src/components/FilesList/ResourceTiles.vue]: Ahh sorry, apparently there is no `getter`, so you need to go via the `state`: `store.state.Files.areFileExtensionsShown`.

And you need to move it into a computed to be reactive then.

Comment (pascalwengerter) [reply packages/web-app-files/src/components/FilesList/ResourceTiles.vue]: Now we should be good

**Pull request merged**